src/aind_analysis_arch_result_access/han_pipeline.py

```diff
@@ -276,6 +271,59 @@ def get_autotrain_table():
     return df_autotrain
 
 
+def get_docDB_table() -> pd.DataFrame:
+    """
+       Load the curriculum data from the behavior json in docDB
+    """
+
+    docdb_client = aind_data_access_api.document_db.MetadataDbClient(
+        host="api.allenneuraldynamics.org", database="metadata_index", collection="data_assets"
+    )
+    sessions = docdb_client.retrieve_docdb_records(
+        filter_query={"name": {"$regex": "^behavior_(?!.*processed).*"}}
```

Comment:
This query will be very slow. To improve performance, we can use 
```json
{
  "session.stimulus_epochs.software.name": "dynamic-foraging-task",
  "data_description.data_level": "raw",  // Use raw is safer because some sessions will fail the processing
  "session.stimulus_epochs": {
    "$elemMatch": {
      "output_parameters.streamlit": { "$exists": true }   // See comment below about the field name "streamlit"
    }
  }
}
```
and using this projection

```json
{"subject_id": 1, "session_start_time": 1, "session.stimulus_epochs.output_parameters.streamlit": 1}
```